api/src/main/java/com/nicknathanjustin/streamercontracts/contracts/ContractService.java

```diff
@@ -42,4 +45,59 @@
      * @param newContractState The new contract state after the result of a vote.
      */
     void setContractState(ContractModel contractModel, ContractState newContractState);
+
+    /**
+     * Gets all contracts in the given state for the given user.
+     *
+     * @param streamer The user to retrieve contracts by.
+     * @param state The state to retrieve contracts by.
+     * @param pageable identifies the page number and pagesize to retrieve
+     * @return a set of all contracts in the given state for the given user.
+     */
+    Page<ContractDto> getContractsForStreamerAndState(UserModel streamer, ContractState state, Pageable pageable);
+
+    /**
+     * Gets all contracts for the given user.
+     *
+     * @param streamer The user to retrieve contracts by.
+     * @param pageable identifies the page number and pagesize to retrieve
+     * @return a set of all contracts for the given user.
+     */
+    Page<ContractDto> getContractsForStreamer(UserModel streamer, Pageable pageable);
+
+    /** Gets all contracts in the given state.
```

Comment:
super nit: spacing is off

Reply:
Sorry, I'm dumb. Could you clarify which line and whats wrong? Its not immediately obvious to me

Reply:
Oh I see. I believe the * are off